README.md

```diff
@@ -140,7 +140,7 @@ private void DataReceivedNormalEvent(object sender, SerialDataReceivedEventArgs
 {
     if (e.EventType == SerialData.WatchChar)
     {
-        // We have our special character at the end of the transmission
+        // We have our special character during the transmission and read was cut short
```

Comment:
Read is never interrupted when the watch char is found. It "just" fires the interrupt to signal that it's present in the input buffer, but the reading goes on.

Reply:
Hopefully made more clear.